### ISSUE TITLE
Slider's aria-disabled attribute does not update

### ISSUE THREAD

**Issue by marcysutton** [angular/material#272]
On a `material-slider` with a disabled expression such as `ng-disabled="color.green < 125"`, `aria-disabled` does not update to `true` along with the presence of the `disabled` attribute. It looks like `updateAriaDisabled()` should be routed through the `$watch` function, but it isn't being fired when the disabled condition is met. 

![Mismatched aria-disabled and disabled attributes on Material slider](https://cloud.githubusercontent.com/assets/1045233/4296177/2165c36c-3df3-11e4-8806-a03a5cb7d1f7.png)

A little more about the disabled attribute: it's not semantically valid on a custom element without extending a native element such as `input` via a Web Component. http://www.w3.org/TR/html5/disabled-elements.html 
I wonder if it makes sense to only use `aria-disabled` for the current state since there is also `disabled` and `ng-disabled`? Attribute city.
